src/main/java/io/vlingo/schemata/Bootstrap.java

```diff
@@ -26,14 +25,15 @@
   private final Server server;
   private final World world;
 
-  public Bootstrap() {
+  public Bootstrap() throws Exception {
     world = World.startWithDefaults("vlingo-schemata");
 
     NoopDispatcher dispatcher = new NoopDispatcher();
-    Journal<String> journal = Journal.using(world.stage(), InMemoryJournalActor.class, dispatcher);
 
-    SourcedTypeRegistry registry = new SourcedTypeRegistry(world);
-    EntryAdapters.register(registry, journal);
+    SchemataObjectStore schemataObjectStore = new SchemataObjectStore("SA", "");
```

Comment:
"SA" with blank password as a bit quick win. Do you want to move them in the configuration file or somewhere else ?

Reply:
@tjaskula Not now. This is ok if we can isolate all the config inside `Bootstrap`.